presto-testng-services/src/main/java/io/prestosql/testng/services/FlakyTestRetryAnalyzer.java

```diff
@@ -49,22 +49,30 @@
     @Override
     public boolean retry(ITestResult result)
     {
+        if (result.isSuccess()) {
+            return false;
+        }
         if (!isEnabled()) {
+            log.info("not retrying; FlakyTestRetryAnalyzer disabled");
             return false;
         }
         Method javaMethod = result.getMethod().getConstructorOrMethod().getMethod();
         if (javaMethod == null) {
+            log.info("not retrying; cannot get java method");
             return false;
         }
         Flaky annotation = javaMethod.getAnnotation(Flaky.class);
         if (annotation == null) {
+            log.info("not retrying; @Flaky annotation not present");
             return false;
         }
         if (result.getThrowable() == null) {
+            log.info("not retrying; throwable not present in result");
             return false;
         }
         String stackTrace = getStackTraceAsString(result.getThrowable());
         if (!Pattern.compile(annotation.match()).matcher(stackTrace).find()) {
+            log.info("not retrying; stacktrace '" + stackTrace + "' does not match pattern '" + annotation.match() + "'");
```

Comment:
make this warn

others can be debug maybe?

Reply:
Debug are disabled by default turning them on is not trivial.